index.html

```diff
@@ -56,7 +56,7 @@ <h2 style="padding-top:0px;">Major Features</h2>
     </div>
     <div class="col-sm-4">
       <h4>Core API</h4>
-      <p>The core <a href="{{ site.baseurl }}/docs/fluo/{{ site.latest_fluo_release }}/">Fluo API</a> supports simple transactional updates using get/set methods.</li>
+      <p>The core <a href="{{ site.baseurl }}/docs/fluo/{{ site.latest_fluo_release }}/">Fluo API</a> supports simple, cross-node transactional updates using get/set methods.</li>
```

Comment:
That's left over html. It should be a `</p>`.  There's another spot on the page where it can be fixed